src/System.Reflection.MetadataLoadContext/ref/System.Reflection.MetadataLoadContext.cs

```diff
@@ -26,4 +26,10 @@ public sealed partial class MetadataLoadContext : System.IDisposable
         public Assembly LoadFromByteArray(byte[] assembly) { throw null; }
         public Assembly LoadFromStream(System.IO.Stream assembly) { throw null; }
     }
+
+    public class PathAssemblyResolver : MetadataAssemblyResolver
```

Comment:
Consider auto-generating the ref assembly: https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/updating-ref-source.md

Reply:
@steveharter, were you able to auto-generate the ref? Do you plan to address that outside of https://github.com/dotnet/corefx/pull/33992?